### PR TITLE
db migration

### PR DESCRIPTION
**Pull request by aphraoh**
#15 

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,6 +45,14 @@
         "spatie/phpunit-watcher": "^1.23",
         "vimeo/psalm": "^5.20"
     },
+    "suggest": {
+        "yiisoft/db-sqlite": "For using with SQLite",
+        "yiisoft/db-mysql": "For using with MySQL",
+        "yiisoft/db-pgsql": "For using with PosgtgreSQL",
```
Comment (vjik) [inline composer.json]:
```suggestion
        "yiisoft/db-pgsql": "For using with PostgreSQL",
```

File: migrations/M240409200600CreateQueueTable.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+declare(strict_types=1);
+
+use Yiisoft\Db\Migration\MigrationBuilder;
+use Yiisoft\Db\Migration\RevertibleMigrationInterface;
+use Yiisoft\Db\Migration\TransactionalMigrationInterface;
+
+final class M240409200600CreateQueueTable implements RevertibleMigrationInterface, TransactionalMigrationInterface
+{
+    private const TABLE_PREFIX = ''; // Add table prefix if required
+    private const QUEUE_TABLE = self::TABLE_PREFIX . 'queue';
```
Comment (vjik) [inline migrations/M240409200600CreateQueueTable.php]:
```suggestion
    private const QUEUE_TABLE = 'queue';
```

If user copy migration, then he can just change table name.

File: migrations/M240409200600CreateQueueTable.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    public function up(MigrationBuilder $b): void
+    {
+        $this->createAssignmentsTable($b);
```
Comment (vjik) [inline migrations/M240409200600CreateQueueTable.php]: Suggest create table here and don't use private method.

File: composer.json [hunk 4]
```diff
@@ -45,6 +45,14 @@
...
+        "yiisoft/db-mssql": "For using with Microsoft SQL Server",
+        "yiisoft/db-oracle": "For using with Oracle",
```
Comment (xepozz) [inline composer.json]: Wouldn't it better mention the only `yiisoft/db`?

Comment (luizcmarin) [reply composer.json]: I don't feel like changing codes. At the moment I'm standardizing all the packages, maybe you've already noticed this.... 🤭 but I think Alexander confirmed to only put the ...db.

File: migrations/M240409200600CreateQueueTable.php (hunk 3 above)
Comment (luizcmarin) [reply migrations/M240409200600CreateQueueTable.php]: Should I do something here?

Comment (luizcmarin) [reply migrations/M240409200600CreateQueueTable.php]: sorry. I thought it was me 😕

File: composer.json (hunk 4 above)
Comment (luizcmarin) [reply composer.json]: sorry. I thought it was me 😕

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**